mariamihaelafainarea/assignment1/Anagrams

```diff
@@ -0,0 +1,165 @@
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Scanner;
+import jdk.nashorn.internal.runtime.ScriptRuntime;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author maria
+ */
+public class Anagrams {
```

Comment:
The approach is generally correct :) 

One observation: the algorithm right now will return false for comparison of "hello world" and "hello    world" (the second one has more than one space between the words). Can you spot why that happens and fix it?

Secondly general comments on the code health:
1. can you remove the debug logging from the methods (it is fine to leave the logs in the main method)
2. make the helper methods in this class private, so that only the method that is expected to be used to solve this problem can be accessed from the outside of this class.
3. add some comments to the methods: one or two sentences that explain what the method does.

Reply:
Yes. I stores me a space like a new word. i have found something to split correctly !